repository/src/main/java/com/epam/esm/configuration/TestRepositoryConfiguration.java

```diff
@@ -4,29 +4,25 @@
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
-import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
 import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 
 import javax.sql.DataSource;
 
 @Configuration
 @Profile("dev")
-@ComponentScan("com.epam.esm.repositoty")
+@ComponentScan("com.epam.esm.repository")
+@EnableTransactionManagement
 public class TestRepositoryConfiguration {
 
     @Bean
-    public DataSource createDataSource(){
+    public DataSource createDataSource() {
```

Comment:
Since you use Spring Boot, you do not need to define @ComponentScan, @EnableTransactionManagement and DataSource bean.

Reply:
removed and fixed